content/company-info-and-process/communication/code_of_conduct.md

```diff
@@ -284,9 +284,31 @@ We regularly implement and update procedures to prevent us from violating intern
 
 ## Raising concerns
 
-If you have a question or concern about a violation of the Code, speak up. Contact your manager, the People Team at people-ops@sourcegraph.com or jonathan.ayers@sourcegraph.com, or the Legal Team at legal@sourcegraph.com or tammy@sourcegraph.com. You can also use Slack, set up a meeting, or call them directly.
+All of us here at Sourcegraph are expected to perform our jobs ethically. If you have any questions or concerns about a violation of our Code of Conduct, we ask that you please come forward.
```

Comment:
@CecilyBlack Even with anonymous reporting, let's leave these internal contacts on the people and legal teams as options. So we can keep the first paragraph the same.